Assets/MixedRealityToolkit/Interfaces/CameraSystem/IMixedRealityCameraSystem.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.CameraSystem
+{
+    public interface IMixedRealityCameraSystem : IMixedRealityEventSystem, IMixedRealityEventSource, IMixedRealityDataProvider
```

Comment:
Could you please comment the interface, like what it should be used for so others can understand why it exists?